scripts/linters/custom_eslint_checks/rules/nested-awaits.js

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Rule to have nested awaits for
+ * await (await browser.switchTo().activeElement()).sendKeys(explanation);
+ * with no exceptions.
+ */
+
+'use strict';
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: `Use nested awaits without any exception for
+       await (await browser.switchTo().activeElement()).sendKeys(explanation);`,
+      category: 'Best Practices',
+      recommended: true,
+    },
+    fixable: null,
+    schema: [],
+    messages: {
+      nestedAwaits: `Only use nested awaits for 
+       await (await browser.switchTo().activeElement()).sendKeys(explanation);`,
+    },
+  },
+
+  create: function(context) {
+    return {
+      CallExpression(node) {
+        if (node && node.callee && node.callee.property &&
+          node.callee.property.name === 'activeElement' &&
+          node.callee.object.callee.property.name === 'switchTo' &&
+          node.callee.object.callee.object.name === 'browser') {
```

Comment:
Instead of so many checks, can we use selector-string to find nodes? (See protractor-practices for ref.)

Reply:
I don't think we can select a whole string like this one `(await browser.switchTo().activeElement())` using string selector. We can only select parts of it like `browser `or `switchTo`. Since each of them (`await`, `browser`,
 `switchTo`, etc) represents a different node, I don't think it's possible to do so. Also, I can't find any example of selecting a whole string in the protractor-practices file.

Reply:
@AdityaDubey0 Can you please check and help @AbhayTrivedi to find string selector?

Reply:
@AdityaDubey0, have you looked at the code yet?

Reply:
Instead of so many checks, you can select the whole string like this:
var elmentAllSelector = (
      'CallExpression[callee.object.callee.object.name=browser]' +
      '[callee.object.callee.property.name=switchTo]' +
      '[callee.property.name=activeElement]');
@AbhayTrivedi can you please try using this selector?
Let me know if you need any help.